pavelib/assets.py

```diff
@@ -190,6 +228,8 @@ def collect_assets(systems, settings):
     for sys in systems:
         sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
 
+    print("\t\tFinished collecting assets.")
```

Comment:
Similar to my other comment for preprocessing assets, please indent this and include a formatted string that says which one it is that finished. (This may actually be the one the has a great time variance between cms and lms.)